cmd/clusterlint/main.go

```diff
@@ -103,12 +103,12 @@ func main() {
 					Name:  "C, ignore-checks",
 					Usage: "run a specific check",
 				},
-				cli.StringFlag{
-					Name:  "n, namespace",
+				cli.StringSliceFlag{
+					Name:  "n, namespaces",
```

Comment:
To preserve backward compatibility you can add `namespace` as another alias here. It will look a little awkward in the help, but I'd like to avoid breaking existing users. I.e.:
```go
Name: "n, namespaces, namespace",
```